notebooks/01_quant_tensor_quant_conv2d_overview.ipynb

```diff
@@ -632,17 +634,17 @@
     {
      "data": {
       "text/plain": [
-       "IntQuantTensor(value=tensor([[[[-9.9000e-03, -7.1000e-03, -4.7000e-03,  5.0000e-03, -1.2300e-02],\n",
-       "          [-8.2000e-03,  8.5000e-03, -1.2000e-03, -1.2500e-02,  4.4000e-03],\n",
-       "          [ 4.3000e-03, -6.3000e-03, -9.4000e-03,  1.0400e-02, -1.2100e-02],\n",
-       "          [ 1.1700e-02, -3.6000e-03,  5.3000e-03, -1.1700e-02, -4.3000e-03],\n",
-       "          [-8.8000e-03,  1.0900e-02, -8.3000e-03, -2.9000e-03,  1.2400e-02]],\n",
+       "IntQuantTensor(value=tensor([[[[ 7.2000e-03, -3.7000e-03,  7.7000e-03, -2.4000e-03, -8.9000e-03],\n",
+       "          [-1.2000e-02, -8.1000e-03,  7.2000e-03, -1.1300e-02, -9.7000e-03],\n",
+       "          [-1.0000e-03,  1.0100e-02,  3.8000e-03, -1.1900e-02,  6.9000e-03],\n",
+       "          [ 8.3000e-03,  1.0000e-04, -6.9000e-03,  3.9000e-03, -5.4000e-03],\n",
+       "          [ 1.1300e-02, -6.0000e-03,  9.7000e-03,  0.0000e+00,  1.0900e-02]],\n",
        "\n",
-       "         [[ 9.3000e-03, -8.5000e-03,  6.5000e-03, -2.7000e-03, -3.4000e-03],\n",
-       "          [-1.0000e-04, -1.1000e-02,  8.3000e-03,  1.9000e-03, -9.8000e-03],\n",
-       "          [ 4.3000e-03, -8.5000e-03,  1.1000e-02,  5.3000e-03,  3.4000e-03],\n",
-       "          [ 8.1000e-03,  9.8000e-03,  6.8000e-03,  1.5000e-03,  6.3000e-03],\n",
-       "          [ 5.7000e-03, -8.5000e-03,  5.2000e-03, -3.0000e-04,  4.9000e-03]]]]), scale=tensor(1.0000e-04), zero_point=tensor(0.), bit_width=tensor(8.), signed_t=tensor(True), training_t=tensor(True))"
+       "         [[-1.0900e-02,  1.1400e-02, -6.4000e-03,  9.2000e-03,  7.1000e-03],\n",
+       "          [-6.0000e-04,  9.2000e-03, -8.5000e-03,  5.0000e-03,  6.5000e-03],\n",
+       "          [-8.3000e-03, -1.2000e-03,  7.4000e-03,  9.2000e-03, -6.0000e-04],\n",
+       "          [-2.1000e-03,  9.5000e-03,  3.0000e-04, -2.9000e-03, -6.5000e-03],\n",
+       "          [-1.1800e-02, -4.8000e-03,  5.4000e-03, -2.5000e-03,  9.0000e-04]]]]), scale=tensor(1.0000e-04), zero_point=tensor(0.), bit_width=tensor(8.), signed_t=tensor(True), training_t=tensor(True))"
```

Comment:
I'm not sure why these values have changed - they all make sense. Maybe the RNG for `torch.randint` has changed in recent PyTorch versions?